lib/getBinaryGltf.js

```diff
@@ -30,7 +30,7 @@ function updateBinaryObject(gltf, pipelineExtras, name, state) {
                     state.currentBinaryView++;
                 }
 
-                var objectSource = object.extras._pipeline.source;
+                var objectSource = new Buffer(object.extras._pipeline.source);
```

Comment:
`Buffer.concat` below didn't like concatting a shader string with a buffer, so I just always wrap the source in a Buffer.